design/noobaa/system-read.md

```diff
@@ -0,0 +1,67 @@
+♜ [Rook NooBaa Design](README.md) /
+# System Read
```

Comment:
Why is it called "System Read"? How about "System Status"?

Reply:
I was just following the CRUD pattern... Do you think its better to change?

Reply:
I changed to `NooBaaSystem Read (Status)`.
Resolving.